rmc/utils/simultaneous_breaks.py

```diff
@@ -167,11 +166,9 @@ def get_sections_to_run(
     .. note::
         This step needs to be run locally due to permissions involved with `parallel_file_exists`.
```

Comment:
Now no longer in use

Reply:
oh you're right -- we don't even use this function anymore. will delete